dapptools/src/utils.rs

```diff
@@ -68,3 +78,55 @@ pub fn open_file(out_path: PathBuf) -> eyre::Result<File> {
         OpenOptions::new().write(true).create_new(true).open(out_path)?
     })
 }
+
+/// Reads the `ETHERSCAN_API_KEY` env variable
+pub fn etherscan_api_key() -> eyre::Result<String> {
```

Comment:
Let's use https://github.com/softprops/envy instead of manual parsing of env vars?

Reply:
since this is a single string value, `envy` would be overhead, I think.
but this looks great to bundle multiple env vars